### ISSUE TITLE
Uncaught exception: +[XCAXClient_iOS sharedClient]: unrecognized selector sent to class

### ISSUE THREAD

**Issue by snowwolflibo** [google/EarlGrey#816]
EarlGrey run error in XCode10.2:
```
Uncaught exception: +[XCAXClient_iOS sharedClient]: unrecognized selector sent to class
```
I have found the newest code has fixed the error:
```
 id XCAXClient;
  if ([XCAXClientClass respondsToSelector:NSSelectorFromString(@"sharedClient")]) {
    XCAXClient = [XCAXClientClass sharedClient];
  } else {
    XCAXClient = [[XCUIDevice sharedDevice] accessibilityInterface];
  }
```
but I can't get the newest version by : 
```
pod 'EarlGrey', '~> 1.15.0' 
```
or 
```
pod 'EarlGrey' 
```
how can I get the newest version?

**Comment by tirodkar** [comment]
Yes, we need to do a release for this change on CocoaPods. The code is fixed at HEAD and you might want to copy the code into your pod if it's local. We will try to get it done this or next week.

https://github.com/google/EarlGrey/pull/808

Should work fine on Simulators though.

**Comment by matcartmill** [comment]
Hey @tirodkar Any update on when the next release will be available that fixes this? The PR has been merged in for nearly 2 months and this is likely affecting a large number of people, myself included.

Would love an update if possible! Thanks!

**Comment by amarama09** [comment]
Hey @tirodkar  Can you please do that release , we are blocked because of this issue https://github.com/google/EarlGrey/pull/808

**Comment by tirodkar** [comment]
To be sure, you need a new CocoaPods / Gem release for this?

**Comment by amarama09** [comment]
@tirodkar  yes , we need a CocoPods/ Gem release

**Comment by tirodkar** [comment]
Release was just done right now. Can you please check if your issues are gone now?

https://cocoapods.org/pods/EarlGrey

**Comment by amarama09** [comment]
Thank you @tirodkar  for doing that release.  We were unblocked the new EarlGrey Version `1.15.1` has fixedhttps://github.com/google/EarlGrey/pull/808

**Comment by matcartmill** [comment]
Thanks for the quick action, @tirodkar.
